sentry-types/src/protocol/v7.rs

```diff
@@ -1737,6 +1737,9 @@ pub struct Span {
     /// Optional extra information to be sent with the span.
     #[serde(default, skip_serializing_if = "Map::is_empty")]
     pub data: Map<String, Value>,
+    /// Optionally HTTP request data to be sent along.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub request: Option<Request>,
```

Comment:
Since you're inserting the request's contents into `data`, this should be removed. (This'll likely be pruned or rejected by relay, if it slips in.)

Reply:
ah thanks!